frappe/commands/utils.py

```diff
@@ -512,6 +512,29 @@ def console(context):
 	IPython.embed(display_banner="", header="", colors="neutral")
 
 
+@click.command('convert-database')
+@pass_context
+def convert_database(context):
+	"convert row_formats to DNAMIC from older formats -- innodb mariadb v10.6.3"
+	site = get_site(context)
+	frappe.init(site=site)
+	frappe.connect()
+
+	information_schema = frappe.qb.Schema("information_schema")
+	queried_tables = frappe.qb.from_(
+		information_schema.tables
+	).select("table_name").where(
+		information_schema.tables.row_format=="Compressed"
+	).run()
+	tables = [x[0] for x in queried_tables]
+
+	for table in tables:
+		frappe.db.sql(f"ALTER TABLE `{table}` ROW_FORMAT=DYNAMIC")
+
+	frappe.db.commit()
```

Comment:
```suggestion
```

Not required for ddl? 

---

Also `db == mariadb` should probably be asserted before running this.